subrootpaths.go

```diff
@@ -141,22 +139,25 @@ func GetSubrootPaths(squareSize uint, idxStart uint, shareCount uint) ([][][]int
 		return nil, srpInvalidShareCount
 	}
 
+	shares := squareSize * squareSize
+
 	// sanity checking
 	if idxStart >= shares || idxStart+shareCount > shares {
 		return nil, srpPastSquareSize
 	}
 
-	// adjust for 0 index
-	shareCount = shareCount - 1
-
-	startRow := int(math.Floor(float64(idxStart) / float64(squareSize)))
-	closingRow := int(math.Ceil(float64(idxStart+shareCount) / float64(squareSize)))
```

Comment:
The linter noticed that the math package is still imported (unnecessarily) btw.

Reply:
Done.